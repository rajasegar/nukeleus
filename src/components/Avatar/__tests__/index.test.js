import React from 'react';
import { renderWithTheme } from '../../../test-utils';

import { Avatar } from '../index';

describe('Avatar component test suite', () => {
  it('Should render Avatar', async () => {
    const { container } = renderWithTheme(<Avatar>AD</Avatar>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
