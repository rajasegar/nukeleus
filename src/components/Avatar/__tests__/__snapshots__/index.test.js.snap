// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component test suite Should render Avatar 1`] = `
.c0 {
  height: 48px;
  width: 48px;
  border-radius: 999px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 > * {
  width: 100%;
  height: 100%;
  border-radius: 999px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<span
  class="c0"
>
  AD
</span>
`;
